Resources/Prototypes/Entities/Objects/Specific/chemistry.yml

```diff
@@ -364,6 +364,16 @@
     - Syringe
     - Trash
 
+- type: entity
+  parent: BaseSyringe
+  id: PrefilledSyringe
+  components:
+  - type: Injector
+    toggleState: Inject
+  - type: Tag
+    tags:
+    - Syringe
```

Comment:
There's no code to re-assign the trash tag when the syringe is used, so this means an empty prefilled syringe won't be trashable but an empty not-prefilled syringe is. This doesn't make much sense.

Reply:
basesyringe should have trashonempty i always thought it did

Reply:
Oh you're completely right. That said I checked the code, and `TrashOnSolutionEmpty` automatically updates the tag on map init, so this still isn't a necessary change.

Reply:
i thought it used componentstartup which is why missing trash tag / having it when it shouldnt made tests fail